cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -436,22 +212,21 @@ ClusterOperationMetadata createCluster(String name, String imageVersion, Map<Str
       if (conf.getSubnet() != null) {
         clusterConfig.setSubnetworkUri(conf.getSubnet());
       } else {
-        clusterConfig.setNetworkUri(network.getSelfLink());
+        clusterConfig.setNetworkUri(nodeClient.getNetwork());
```

Comment:
Any specific reason to name it `nodeClient` here and `networkClient` in `fromConf`?